### DISCUSSION TITLE
I am facing issue while converting ICO to PNG file

### DISCUSSION THREAD

**Discussion by JaydipGolviyaTR** [ImageMagick/ImageMagick#5051] [Help]
I am facing some issue while converting ICO to PNG file only on some of ICO files 
Need a help for that

I have attached ICO file which is failed in converting 

[favicon.zip](https://github.com/ImageMagick/ImageMagick/files/8504840/favicon.zip)

Please help me out

**Comment by dlemstra** [comment]
What is the issue that you are experiencing?

**Reply by JaydipGolviyaTR** [reply]
Like when we converting ICO to PNG image so it will generating PNG image file with 0KB size
We can't open it because it showing corrupted file

**Comment by snibgo** [comment]
What version of IM, on what platform? What is your command?

With IM v7.1.0-4 on Windows 8.1, the command ...
```
magick favicon.ico x.png
```
... works fine, creating three valid PNG files, named x-0.png, x-1.png and x-2.png.

**Reply by JaydipGolviyaTR** [reply]
I am using java platform with v1.4.0 of the library
Attached snippet code which I performed
 
`IMOperation op = new IMOperation();`
`op.addImage(imageToConvert.getAbsolutePath());  `
`op.addImage(convertedImage.getAbsolutePath()); `
`ConvertCmd convert = new ConvertCmd();`
`convert.run(op);`

imageToConvert param is ICO file path
convertedImage param is PNG file path

**Comment by snibgo** [comment]
Sorry, I know nothing about Java.
